arch/arm/boot/dts/overlays/mpu6050-overlay.dts

```diff
@@ -0,0 +1,24 @@
+// Definitions for MPU6050
+/dts-v1/;
+/plugin/;
+
+/ {
+        compatible = "brcm,bcm2708";
+
+        fragment@0 {
+                target = <&i2c1>;
+                __overlay__ {
+                        #address-cells = <1>;
+                        #size-cells = <0>;
+                        status = "okay";
+                        clock-frequency = <400000>;
+
+                        inv-mpu6050@68 {
```

Comment:
I don't think the `inv-` prefix adds anything - the dtbindings documentation just uses `mpu6050@68`.